src/components/helper/auth.js

```diff
@@ -0,0 +1,99 @@
+import decode from 'jwt-decode';
+
+export default class AuthService {
```

Comment:
If you have your `AuthService` class setup to handle authentication then you should use it for your `App` component (`handleLogout`)